### PR TITLE
Log cluster metrics in Runtime Agent

### PR DESCRIPTION
**Pull request by franpog859**
**Description**

Changes proposed in this pull request:

- Log cluster metrics in Runtime Agent so FluentBit can pass them further

**Related issue(s)**

_none (!)_

### CODE REVIEW

File: components/compass-runtime-agent/cmd/main.go [hunk 1]
```diff
@@ -114,10 +113,40 @@ func main() {
...
+	metricsLogger, err := newMetricsLogger(options.MetricsLoggingTimeInterval)
+	if err != nil {
+		log.Error(errors.Wrap(err, "Unable to create metrics logger"))
 	}
+
+	runManagerAndLoggerConcurrently(mgr, metricsLogger)
+	os.Exit(1)
+}
+
+func runManagerAndLoggerConcurrently(manager manager.Manager, logger metrics.Logger) {
```
Comment (koala7659) [inline components/compass-runtime-agent/cmd/main.go]: I suppose it is possible that logger maybe passed as nil here - after newMetricsLogger function failed (for example when creation of Resource fetcher has failed).  In such a case logger.Log(quitChannel) (line 146) will crash

Comment (franpog859) [reply components/compass-runtime-agent/cmd/main.go]: Good catch! Thanks, I'll fix it ;>

**Pull request merged**